backend_plugin/src/main/evaluators/Evaluator.java

```diff
@@ -63,7 +63,7 @@ private void initializeListeners(ITextEditor textEditor) {
 		this.document = doc;
 
 		// Add a DocumentChangesTracker to the document
-		DocumentChangesTracker docTracker = new DocumentChangesTracker(this);
+		DocumentChangesListener docTracker = new DocumentChangesListener(this);
```

Comment:
Is there a better name for this as well? Instead of docTracker. documentListener? docChangeListener? This keeps the consistency throughout.

Reply:
Updated the PR for this comment as well.